<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigurationDialog</class>
 <widget class="QWidget" name="ConfigurationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>536</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QFrame { background-color : white }</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>241</width>
     <height>81</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <widget class="QLineEdit" name="lineSecondPlayerName">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>39</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>1. játékos:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>2. játékos:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineFirstPlayerName">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>9</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>10</y>
     <width>131</width>
     <height>81</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <widget class="QSpinBox" name="spinCols">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>40</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Oszlop:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinRows">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>11</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>46</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Sor:</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="cardFrame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>381</width>
     <height>391</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Kártyacsomag:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboCardPack">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>10</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Kártyák száma:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelCardCount">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>50</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>361</width>
      <height>301</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="cardLayout"/>
   </widget>
  </widget>
  <widget class="QPushButton" name="buttonOk">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>490</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonCancel">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>490</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Cancel</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineFirstPlayerName</tabstop>
  <tabstop>lineSecondPlayerName</tabstop>
  <tabstop>spinRows</tabstop>
  <tabstop>spinCols</tabstop>
  <tabstop>comboCardPack</tabstop>
  <tabstop>buttonCancel</tabstop>
  <tabstop>buttonOk</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
